packages/quantic/force-app/main/default/lwc/quanticSmartSnippetAnswer/quanticSmartSnippetAnswer.js

```diff
@@ -69,6 +69,8 @@ export default class QuanticSmartSnippetAnswer extends LightningElement {
     const disabledCSSClass = 'inline-link--disabled';
     this.inlineLinks.forEach((link) => {
       if (link?.href) {
+        link.target = '_blank';
```

Comment:
@MehdiMitiche should we put this outside of the "if" ? Not sure what use case we have where the href is null.